src/platforms/blackpill-f401ce/platform.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of the Black Magic Debug project.
+ *
+ * Copyright (C) 2011  Black Sphere Technologies Ltd.
+ * Written by Gareth McMullin <gareth@blacksphere.co.nz>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* This file provides the platform specific declarations for the blackpill-f401ce implementation. */
+
+#ifndef PLATFORMS_BLACKPILLF401CE_PLATFORM_H
```

Comment:
Hyphens should be translate into _'s here which helps with readability - `PLATFORMS_BLACKPILL_F401CE_PLATFORM_H`

Likewise for the other platform.h headers here